mgmtd/mgmt_be_adapter.c

```diff
@@ -34,55 +35,31 @@
  */
 struct mgmt_be_xpath_map {
 	char *xpath_regexp;
```

Comment:
Maybe rename it to `xpath_prefix` to reflect the reality.

Reply:
yes that's right.